pkg/util/parse_test.go

```diff
@@ -57,13 +57,9 @@ const yamlTestPathSeparators = `
 pathFromLinux:
     - id: "here/dir/file.id"
     - name: "here\\dir/file.name"
```

Comment:
@UnseenWizzard can you tell me what exactly we expect to happen here? 

Do expect the name to end up as `here/dir/file.name` or `here\\dir/file.name`?